projects/docs-tools-showcase/src/app/home/mock-documentation.json

```diff
@@ -10,29 +10,27 @@
     "SkyColorpickerMessage": "interface-skycolorpickermessage",
     "SkyColorpickerOutput": "interface-skycolorpickeroutput",
     "SkyColorpickerResult": "interface-skycolorpickerresult",
-    "SkyColorpickerRgba": "interface-skycolorpickerrgba",
-    "SkyColorpickerUtils": "class-skycolorpickerutils"
+    "SkyColorpickerRgba": "interface-skycolorpickerrgba"
```

Comment:
Were these changes intentional?

Reply:
It reflects the new dev-infra setup, not necessarily related to code-examples, but is related to Sky UX 8.